jpa/spi/pom.xml

```diff
@@ -53,12 +53,14 @@
         <dependency>
             <groupId>jakarta.persistence</groupId>
             <artifactId>jakarta.persistence-api</artifactId>
+            <version>${version.jakarta.persistence}</version>
```

Comment:
This isn't needed; it's controlled by the bom.

Reply:
Removed.